script/dev/create-config.sh

```diff
@@ -11,3 +11,5 @@ fi
 
 docker run --rm -it -v "$PWD":/working broadinstitute/dsde-toolbox \
   render-templates.sh local $VAULT_TOKEN
+
+cp src/main/config/dsde-dev/config.json target
```

Comment:
I don't feel super strong about this, but I'd prefer to add a src/main/config/dsde-local directory, so that this script only concerns itself with the "local" environment. I realize the config in dsde-local would point at https://shibboleth.dsde-dev.broadinstitute.org.